pkgs/tools/backup/dar/default.nix

```diff
@@ -47,20 +45,18 @@ stdenv.mkDerivation rec {
     "--enable-threadar"
   ];
 
+  hardeningDisable = [ "format" ];
+
   postInstall = ''
     # Disable html help
     rm -r "$out"/share/dar
   '';
 
-  enableParallelBuilding = true;
```

Comment:
Why was this removed? I don't think it's enabled by default unless you use `cmake`, `meson`, `ninja` or something like it

Reply:
Oh! I thought that option was always enabled by default and that's why I removed it. Nice catch.